harvard-lts/CURIOSity
# GitHub push 11 1

**Github Push 11-1*
* * *


# What does this Pull Request do?
Fixes the logic problem of not having a cascade delete in harvesters table when an exhibit is purged

# How should this be tested?

Run this branch locally, create an exhibit and do a harvest. Look at your local db and see that the harvester row with the exhibit id appears. Then delete the exhibit and look at your db again to see that the corresponding harvest line(s) are gone (so if your exhibit had an id of 2, all rows with 2 should be gone from harvesters when it is deleted)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
